packages/api/lib/writeRecords/write-granules.js

```diff
@@ -579,9 +580,10 @@ const _writeGranuleRecords = async (params) => {
       `Completed write operation to PostgreSQL for granule %j. Record cumulus_id in PostgreSQL: ${writePgGranuleResult.pgGranule.cumulus_id}.`,
       postgresGranuleRecord
     );
+    // TODO: need to log either limitedUpdateApiGranuleRecord || apiGranuleRecord
     log.info(
       'Completed write operation to DynamoDb for granule %j',
-      apiGranuleRecord
+      limitedUpdateApiGranuleRecord || apiGranuleRecord
```

Comment:
This is neat. I probably would have done this differently and it would have been messy 😄